extensions/intermittent-fasting/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "intermittent-fasting",
+  "title": "Intermittent Fasting",
+  "description": "Tracks intermittent fasting",
+  "icon": "icon.png",
+  "author": "pernielsentikaer",
+  "categories": [
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "intermittent-fasting",
+      "title": "Track Fasting",
+      "description": "List intermittent fasting",
+      "mode": "view"
+    },
+    {
+      "name": "menubar",
+      "title": "Fasting Status",
+      "description": "Show current fasting status in menubar",
+      "mode": "menu-bar",
+      "interval": "1m",
+      "preferences": [
+        {
+          "name": "menubarTitle",
+          "type": "textfield",
+          "required": false,
+          "title": "Menubar Title Format",
+          "description": "Customize the menubar title. Available variables: {timeLeft}, {percentage}, {time}",
+          "default": "{percentage}"
+        },
+        {
+          "name": "idleDisplay",
+          "type": "dropdown",
+          "required": false,
+          "title": "Idle Display",
+          "description": "Choose how the menubar appears when no timer is running",
+          "data": [
+            {
+              "title": "None",
+              "value": "none"
+            },
+            {
+              "title": "Icon Only",
+              "value": "icon"
+            },
+            {
+              "title": "Icon with Text",
+              "value": "full"
+            },
+            {
+              "title": "Show Eating Window",
+              "value": "eating"
+            }
+          ],
```

Comment:
How about adding a checkbox option to allow users to switch `Show Eating Window on Idle Display` for all options? Then we have:

- Icon Only (show eating window)
- Icon Only (no eating window)
- Icon with Text (show eating window)
- Icon with Text (no eating window)

Reply:
That's a good idea!